account_asset_management_xls/report/__init__.py

```diff
@@ -1,2 +1,10 @@
-# -*- coding: utf-8 -*-
-from . import account_asset_report_xls
+
+try:
```

Comment:
```suggestion
# The whole python file is ignored when the dependency is missing
# because it contains an old-style rml_parse report which would
# be loaded anyway
try:
```